### PR TITLE
translate and fix RTL layout (WIP)

### PR DESCRIPTION
**Pull request by ca3tiel**
Translate and fix RTL layout

### CODE REVIEW

File: app/src/main/res/layout/activity_welcome.xml [hunk 1]
```diff
@@ -37,12 +37,12 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:fontFamily="@font/museo_san_rounded700"
-        android:text="Best iranian VPN"
+        android:text="@string/best_iranian_vpn"
```
Comment (kamrava) [inline app/src/main/res/layout/activity_welcome.xml]: Please change iranian name to something else!

File: app/src/main/res/values-ar/strings.xml [hunk 2]
```diff
@@ -376,4 +376,53 @@
...
+    <string name="plan">Plan</string>
+    <string name="expiration_date">Expiration date</string>
+    <string name="dataleft">DataLeft</string>
+    <string name="title">Title</string>
+    <string name="enter_your_title">Enter your title</string>
+    <string name="message">Message</string>
+    <string name="enter_your_message">Enter your Message</string>
+    <string name="send">Send</string>
+    <string name="exit_account">Exit Account</string>
 </resources>
```
Comment (kamrava) [inline app/src/main/res/values-ar/strings.xml]: Add new line

File: app/src/main/res/values-be/strings.xml [hunk 3]
```diff
@@ -106,4 +106,53 @@
...
+    <string name="plan">Plan</string>
+    <string name="expiration_date">Expiration date</string>
+    <string name="dataleft">DataLeft</string>
+    <string name="title">Title</string>
+    <string name="enter_your_title">Enter your title</string>
+    <string name="message">Message</string>
+    <string name="enter_your_message">Enter your Message</string>
+    <string name="send">Send</string>
+    <string name="exit_account">Exit Account</string>
 </resources>
```
Comment (kamrava) [inline app/src/main/res/values-be/strings.xml]: Same here

File: hs_err_pid31952.log
Comment (kamrava) [inline hs_err_pid31952.log:1]: Do we really need this file?

File: hs_err_pid6816.log
Comment (kamrava) [inline hs_err_pid6816.log:1]: Same as [this](https://github.com/ca3tiel/NekoBoxForAndroid/pull/3/files#r1398408795)

File: replay_pid31952.log
Comment (kamrava) [inline replay_pid31952.log:1]: Same as [this](https://github.com/ca3tiel/NekoBoxForAndroid/pull/3/files#r1398408795)

File: app/src/main/res/layout/activity_welcome.xml (hunk 1 above)
Comment (ca3tiel) [reply app/src/main/res/layout/activity_welcome.xml]: chaged to :
```
<string name="welcome_title">Best iranian VPN</string>
```

File: app/src/main/res/values-ar/strings.xml (hunk 2 above)
Comment (ca3tiel) [reply app/src/main/res/values-ar/strings.xml]: all fixed

File: app/src/main/res/values-be/strings.xml (hunk 3 above)
Comment (ca3tiel) [reply app/src/main/res/values-be/strings.xml]: fix

**Pull request merged**